### PR TITLE
Pep8 codebase styling

### PR DESCRIPTION
**Pull request by drewstone**
# Issue
https://github.com/ethereum/cbc-casper/issues/175

# Implementation
Current only styling/formatting changes.

### CODE REVIEW

File: casper/plot_tool.py [hunk 1]
```diff
@@ -194,7 +199,8 @@ def make_thumbnails(self, frame_count_limit=IMAGE_LIMIT, xsize=1000, ysize=1000)
         iterator = 0
         for image in images:
             image.thumbnail(size, Image.ANTIALIAS)
-            image.save(self.thumbnail_path + str(1000 + iterator) + "thumbnail.png", "PNG")
+            image.save(self.thumbnail_path + str(1000 +
+                                                 iterator) + "thumbnail.png", "PNG")
```
Comment (djrtwo) [inline casper/plot_tool.py]: I would prefer
```python
image.save(self.thumbnail_path + str(1000 + iterator) +
           "thumbnail.png", "PNG")
```

File: casper/protocols/binary/binary_estimator.py [hunk 2]
```diff
@@ -5,8 +5,10 @@
 def get_estimate_from_latest_messages(latest_bets, default=None):
     """Picks the highest weight estimate (0 or 1) given some latest bets."""
 
-    zero_weight = sum(v.weight for v in latest_bets if latest_bets[v].estimate == 0)
-    one_weight = sum(v.weight for v in latest_bets if latest_bets[v].estimate == 1)
+    zero_weight = sum(
```
Comment (djrtwo) [inline casper/protocols/binary/binary_estimator.py]: generally codebase has been putting the `)` on a new line for readability

```python
one_weight = sum(
    v.weight for v in latest_bets if latest_bets[v].estimate == 1
)

Comment (djrtwo) [reply casper/protocols/binary/binary_estimator.py]: Comment stands for the rest of the PR

File: casper/plot_tool.py (hunk 1 above)
Comment (drewstone) [reply casper/plot_tool.py]: definitely, I'll go through and fix. Thanks!

**Comment by drewstone** [comment]
@djrtwo We can add something to `Contributing.md`. I presume it is derived from Pep8 styling. Maybe we can add a lint template for using a plugin like Flake8? You're right that most of the changes were from 80 vs 100 character break so I've changed that in my linter below.

```
{
    // run flake8 lint on file saving
    "lint_on_save": true,
    // run flake8 lint on file loading
    "lint_on_load": false,

    // run lint in live mode: lint file (without popup) every XXX ms
    // please, be careful: this may cause performance issues on ST2
    "live_mode": false,
    // set live mode lint delay, in milliseconds
    "live_mode_lint_delay": 1000,

    // set ruler guide based on max line length setting
    "set_ruler_guide": false,

    // popup a dialog of detected conditions?
    "popup": false,
    // highlight detected conditions?
    "highlight": true,

    // highlight type:
    // - "line" to highlight whole line
    // - "error" to highlight error only
    "highlight_type": "error",

    // color values to highlight detected conditions
    "highlight_color_critical": "#981600",
    "highlight_color_error": "#DA2000",
    "highlight_color_warning": "#EDBA00",

    // show a mark in the gutter on all lines with errors/warnings:
    // - "dot", "circle" or "bookmark" to show marks
    // - "theme-alpha", "theme-bright", "theme-dark", "theme-hard" or "theme-simple" to show icon marks
    // - "" (empty string) to do not show marks
    "gutter_marks": "theme-simple",

    // report successfull (passed) lint
    "report_on_success": false,

    // blink gutter marks on success (will not blink with live mode check)
    // this icon is not depends on 'gutter_marks' settings
    // please, be careful: this may cause performance issues on ST2
    "blink_gutter_marks_on_success": true,

    // load global flake8 config ("~/.config/flake8")
    "use_flake8_global_config": true,
    // load per-project config (i.e. "tox.ini", "setup.cfg" and ".pep8" files)
    "use_flake8_project_config": true,

    // set python interpreter (lint files for python >= 2.7):
    // - 'internal' for use internal Sublime Text interpreter (2.6)
    // - 'auto' for search default system python interpreter (default value)
    // - absolute path to python interpreter for define another one
    //   use platform specific notation, i.e. "C:\\Anaconda\\envs\\py33\\python.exe"
    //   for Windows or then "/home/whatever/pythondist/python" for Unix
    "python_interpreter": "auto",

    // list of python built-in functions (like '_')
    "builtins": [],

    // turn on pyflakes error lint
    "pyflakes": true,
    // turn on pep8 error lint
    "pep8": true,
    // turn on pydocstyle error lint
    "pydocstyle": true,
    // turn on naming error lint
    "naming": true,
    // turn on debugger error lint
    "debugger": true,
    // turn on import order error lint
    "import-order": false,
    // import order style: "cryptography" or "google"
    // See also: https://github.com/public/flake8-import-order#configuration
    "import-order-style": "cryptography",
    // turn off complexity check (set number > 0 to check complexity level)
    "complexity": -1,

    // set desired max line length
    "pep8_max_line_length": 100,

    // select errors and warnings (e.g. ["E", "W6"])
    "select": [],
    // skip errors and warnings (e.g. ["E303", "E4", "W"])
    "ignore": [],

    // files to ignore, for example: ["*.mako", "test*.py"]
    "ignore_files": []
}
```

**Pull request closed without merging**